openquake/calculators/extract.py

```diff
@@ -230,3 +230,20 @@ def extract_aggdamages(dstore, loss_type, *tags):
     else:
         raise KeyError('No damages found in %s' % dstore)
     return filter_agg(dstore, losses, tags)
+
+
+@extract.add('aggcurves')
+def extract_aggcurves(dstore, loss_type, *tags):
+    """
+    Aggregate loss curves of the given loss type and tags for
+    event based risk calculations.
+
+    :returns:
+        array of shape (S, P), being P the number of return periods
+        and S the number of statistics
```

Comment:
It would be useful to extract with the same call, as attributes, also the names of stats and the list of return periods.